### ISSUE TITLE
[BUG]: Very Long Black Screens

### ISSUE THREAD

**Issue by Emplehod** [yuliskov/SmartTube#3814]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/yuliskov/SmartTube/issues) or [closed](https://github.com/yuliskov/SmartTube/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://github.com/yuliskov/SmartTube#faq) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.

### Affected version

22.95

### Device Type

Smart TV/Box

### Affected Android

Nvidia Shield Pro 9.1.1 (33.2.0.157)

### Steps to reproduce the bug

Play any video.
Escape any video.

### Actual behavior

Starting videos will display a black screen for 2 seconds, then the video will appear for 1/2 second, then go black again. About 2 seconds later, audio will play, and after 2-3 seconds more, video starts.

Exiting a video displays a black screen for about 4-5 seconds before returning to the menu. YouTube app is instant, no delays.

### Additional information

I appreciate you creating this app for free use. I'm sure it takes a lot of time, but since the last few updates, this has occurred. It is less annoying to use the stock app and skip (or watch) the ads.

**Comment by ACR-Jeff** [comment]
I have the same issue on a Nvidia Shield 2017 in 23.13 Beta, Black screen followed by a flash of the video, Then a few seconds later a black screen again for a few seconds some times, Yet other times it will last a minute, Depending on if I just started the app, Etc. 
I have tried clearing cache and data, Re-login, Same thing.

**Comment by ACR-Jeff** [comment]
Tested with 22.95 Stable, Uploaded videos seem to play right away, Active Live streams still have the black screen delay.

**Comment by Emplehod** [comment]
It only started about 2-3 updates ago for me. I figured it was maybe YouTube making it more difficult to remove ads. It certainly isn't a Shield issue. The stock YouTube app doesn't do it. It also occurs (for me) on every video, stream or uploaded.

**Comment by ACR-Jeff** [comment]
Yes that's about when it started for me as well, I have waited for an update, Yet nothing has fixed it on my end. For me it's some times at random, But the majority of the time it happens, I also have noticed ST being really laggy/sluggish at times in navigation, Etc.

**Comment by ACR-Jeff** [comment]
The long black screen delay and buffering has been fixed for me with 23.68 Beta, Thank you.

**Comment by ACR-Jeff** [comment]
The Long black screen delay with buffering appeared again on the latest beta, 
I browsed the settings and noticed the "Network engine" was changed from what I have originally set it to, I usually have it set to either default or Cronet, I have never set it to OkHttp because it's so slow as it states in the settings menu. I even checked by restoring a recent backup, Which was set to Cronet, Maybe check to make sure yours wasn't changed at some point
Settings > Player > Network engine
Changing it back improved the startup of a video
